libnetdata/eval/eval.c

```diff
@@ -1077,6 +1077,7 @@ int expression_evaluate(EVAL_EXPRESSION *expression) {
         // although there is an unknown variable
         // the expression was evaluated successfully
         expression->error = EVAL_ERROR_OK;
+        expression->result = NAN;
```

Comment:
This change is from #6984

I think it shouldnt be there

Reply:
Yesterday I found another place to put this result, but I did not have time to test the impact of it in the current results, I will do this during the weekend.
Thanks!